app/controllers/application_controller.rb

```diff
@@ -2,4 +2,7 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  def after_sign_out_path_for(resource_or_scope)
```

Comment:
can include after sign in path also that should be on root path, and after signout should ideally have login page